requirements.txt

```diff
@@ -22,4 +22,5 @@ langchain==0.1.5
 langchainhub==0.1.14
 python-dotenv
 wikipedia_sections
-vllm
\ No newline at end of file
+vllm
+loguru
```

Comment:
According to this [commit](https://github.com/opentensor/bittensor/commit/9d7bbbef957e9ffa830a1cbbb8dbd4a91af96feb), bittensor does not use loguru anymore. It would be nice to adopt this on prompting itself, following best practices of python. Issue created #200